extensions/banner/deployment/src/main/java/io/quarkus/banner/deployment/BannerConfig.java

```diff
@@ -0,0 +1,14 @@
+package io.quarkus.banner.deployment;
+
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+@ConfigRoot(name = "banner", phase = ConfigPhase.BUILD_TIME)
+public class BannerConfig {
+    /**
+     * The path of the banner which could be provided by user
```

Comment:
I think it would be nice to have an example here showing how a user can put a custom banner (and where they need to add this file).

Other than this, I'm good with the PR

Reply:
I am thinking of telling the user that adding `my_banner.txt` in `src/main/resources` then just using `quarkus.banner.path=my_banner.txt`

Reply:
Wouldn't that be good for documentation purpose ? A proper document would be more clear IMO

Reply:
The good thing is that these config properties generate documentation automatically (see other config objects for example).
So it would be a nice first step to have a nice javadoc for the property since Quarkus users will know where to see the documentation.

Reply:
how to do `javaDoc` then ?

Reply:
See for `MongoClientConfig` for example.

Reply:
Could you please tell in which module ? :sweat_smile:

Reply:
`io.quarkus.mongodb.runtime.MongoClientConfig` in `extensions/mongodb-client/runtime`. Just look at `connectionString` string where a detailed javadoc has been added